rootfs/etc/nginx/lua/balancer.lua

```diff
@@ -38,6 +38,10 @@ end
 
 local function get_current_lb_alg()
   local backend = get_current_backend()
+  if not backend then
+    return nil
```

Comment:
is not necessary for this PR, but I thought its small enough change I can include here

Reply:
:+1: